CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+Thanks for wanting to contribute!
+
+Feel free to create issues or make pull requests, we'll try to quickly review them.
+
+If you need to reach out to us, find a relevant [issue](https://github.com/killercup/assert_cli/issues) or open a new one.
+
+# Issues
+
+Some helpful pieces of information when reporting issues
+* assert_cli version
+* rust version
+* OS and version
+
+# Pull Requests
+
+## Project Ideas
+
+If you're looking for things to do check out the [open issues](https://github.com/killercup`/assert_cli/issues).
+Or take a grep through [all TODO comments](https://github.com/killercup/assert_cli/search?q=TODO) in the code and feel free to help us out there!
+
+## Best Practices
+
+We appreciate your help as-is.  We'd love to help you through the process for contributing.  We have some suggestions to help make things go more smoothly.
```

Comment:
Is double-spacing a thing people do?

Reply:
My understanding is its an archaic convention originating with type writers and obviously I've not broken myself of it :)